### PR TITLE
Add logging update

### PR DESCRIPTION
**Pull request by dlapiduz**
Some details on where and how to use it...

### CODE REVIEW

**Comment by dlapiduz** [comment]
@mogul @afeld @LinuxBozo :wave:

File: _posts/2016-02-12-logs-update.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+## New logging service
+
+The cloud.gov recently launched the shared logging service. You can access it today at https://logs.cloud.gov.
+
+Given the common the need for extended and compliant logging, we decided to build it as a service. Now, every cloud.gov  application will have their logs stored automatically, zero setup required.
+
+To use it, visit https://logs.cloud.gov and use your cloud.gov credentials to log in. You can access both application logs and platform events, all of which will be persisted for 180 days.
+
+Now that we have a shared service for logs, the ELK service will be deprecated. If you are still using it, we recommend you delete the instance whenever is convenient.
```
Comment (afeld) [inline _posts/2016-02-12-logs-update.md]: Can we include general instructions for this?

**Comment by afeld** [comment]
Made some minor tweaks to the wording, but otherwise looks good!

**Comment by dlapiduz** [comment]
I will leave it for @mogul then

**Comment by mogul** [comment]
I'm out today, but if you agree with my changes then :shipit:.

**Pull request merged**